### PR TITLE
Template/p6 container

### PR DESCRIPTION
**Pull request by vuongc**
Added back the dependencies to `@storybook/addon-knobs` because some stories are still using it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "@storybook/addon-actions": "^6.0.0-rc.11",
     "@storybook/addon-controls": "^6.0.0-rc.11",
     "@storybook/addon-docs": "^6.0.0-rc.11",
+    "@storybook/addon-knobs": "^5.3.19",
```
Comment (vincentv) [inline package.json:55]: We don't use knobs anymore.

File: src/components/molecules/p6-container/p6-container.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+:host {
+  display: block;
+  padding: $size-7;
```
Comment (vincentv) [inline src/components/molecules/p6-container/p6-container.scss:3]: Maybe we should create a variable for the padding of the container.

File: src/components/molecules/p6-container/p6-container.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import readme from "./readme.md";
+
+import { DefaultStory } from "./p6-container.stories.ts";
+
+<Meta title="Templates/Container" component="p6-container" />
+
+# Container
+
+<Preview>
+  <Story name="Default" height="336px">
```
Comment (jswale) [inline src/components/molecules/p6-container/p6-container.stories.mdx:12]: missing the parameters for args, parameters and argType (cf other stories files)

File: src/components/molecules/p6-container/p6-container.stories.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { getComponent, makeStory } from "../../../shared/storybook/stories";
+
+const getStoryField = (text: string): string =>
+  getComponent("p6-container", text);
+
+export const DefaultStory = makeStory<{
+  text: string;
+}>({
+  args: {
+    text: "Container exemple",
```
Comment (jswale) [inline src/components/molecules/p6-container/p6-container.stories.ts:10]: example

File: src/components/molecules/p6-container/test/p6-container.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { newSpecPage } from "@stencil/core/testing";
+import { P6Container } from "../p6-container";
+
+describe("p6-container", () => {
+  it("renders", async () => {
+    const page = await newSpecPage({
+      components: [P6Container],
+      html: `<p6-container></p6-container>`,
+    });
+    expect(page.root).toEqualHtml(`
```
Comment (jswale) [inline src/components/molecules/p6-container/test/p6-container.spec.tsx:10]: toMatchSnapshot

File: package.json (hunk 1 above)
Comment (vuongc) [reply package.json:55]: See comment of the pr

File: src/components/molecules/p6-container/test/p6-container.spec.tsx (hunk 5 above)
Comment (vuongc) [reply src/components/molecules/p6-container/test/p6-container.spec.tsx:10]: Snapshot added

File: src/components/molecules/p6-container/p6-container.stories.mdx (hunk 3 above)
Comment (vuongc) [reply src/components/molecules/p6-container/p6-container.stories.mdx:12]: props has been added

File: src/components/molecules/p6-container/p6-container.scss (hunk 2 above)
Comment (vuongc) [reply src/components/molecules/p6-container/p6-container.scss:3]: `$container-padding: $size-7` added and used

File: src/components/molecules/p6-container/p6-container.stories.ts (hunk 4 above)
Comment (vuongc) [reply src/components/molecules/p6-container/p6-container.stories.ts:10]: done

File: package.json (hunk 1 above)
Comment (jswale) [reply package.json:55]: Now you can remove the knobs :D

Comment (vuongc) [reply package.json:55]: Yeahh

Comment (vuongc) [reply package.json]: It's been removed

**Pull request merged**